client/pages/chef.jsx

```diff
@@ -95,13 +103,26 @@ class ChefProfile extends React.Component {
       });
   }
 
+  handleClickSave() {
+    fetch(`/api/userProfile/${this.props.chefId}/${this.props.userId}`, {
+      method: 'POST',
+      headers: {
```

Comment:
Instead of sending the userId in the url path, you should send the logged in user's token in the `x-access-token` header of your fetch. that way clients can't lie about who they are.